sdks/python/apache_beam/io/gcp/bigquery_file_loads_test.py

```diff
@@ -615,7 +615,6 @@ def setUp(self):
                  self.dataset_id, self.project)
 
   @attr('IT')
-  @unittest.skip('BEAM-8842: Disabled due to reliance on old retry behavior.')
```

Comment:
Should we skip both versions? Is this version not timing out?

Reply:
No, just this version should be skipped. I double checked the logs in the original bug report.